exercises/numbers/isPrime/isPrime.js

```diff
@@ -11,6 +11,16 @@
  * @returns {boolean} True if num is prime and false otherwise
  */
 function isPrime(num) {
+  if (num <= 1 ) {
+    return false
+  }
+  for (i = 2; i < num; i++) {
```

Comment:
This works, but you can stop looking much sooner. For example, if I were to ask you whether `101` is prime, would you check to see whether `100` divided `101`?

The first `i` you find such that `num % i === 0` will be the smallest prime divisor of `num`. What's the largest the smallest prime divisor of a non-prime number can be? Once you're beyond that point you know you won't find any prime divisors and therefore `num` itself must be prime.

A thing I wrote up if you're interested in the math: http://smallest-prime.surge.sh/smallest-prime.pdf

You can kind of get an intuitive sense by thinking through how you'd do it by hand. When would you stop looking for larger potential divisors?